### DISCUSSION TITLE
speedup pg_dump and restore

### DISCUSSION THREAD

**Discussion by gillbates** [prodrigestivill/docker-postgres-backup-local#87] [Q&A] [answered]
Hi Team,

thanks for this wonderful docker. It really simplify the things.
However the current pg_dump command is very slow,
can we use 

`pg_dump -j 8 -Fd -f /tmp/newout.dir fsdcm_external`

and then output with a tar.gz file?

https://stackoverflow.com/questions/2094963/postgresql-improving-pg-dump-pg-restore-performance

**Comment by prodrigestivill** [comment]
Is this issue not a duplicate of #33 ?

**Comment by gillbates** [comment]
nope, i mean use 'j 8' to speed up the back and restore process?
maybe we could 
1, use this j 8 and -Fd as the default way, and then compress the whole directory into a tgz file fom simplicity?
2, we use our built-in script to unzip the tgz file and pg_restore ?

**Comment by prodrigestivill** [comment, marked as answer]
If I understand correctly you can use `POSTGRES_EXTRA_OPTS` env variable for that, is that right?
